client/services/PlanningStoreService.js

```diff
@@ -15,10 +15,13 @@ PlanningStoreService.$inject = [
     'notifyConnectionService',
     'userList',
     'desks',
+    'templates',
     'metadata',
     'session',
     'deployConfig',
     'gettext',
+    'content',
```

Comment:
Is `content` used in any actions? If not we can remove `content` from the dependencies.

Reply:
indeed, was forgotten from a previous version, will update, thanks!